lib/lightning/workflows/edge.ex

```diff
@@ -30,6 +30,16 @@ defmodule Lightning.Workflows.Edge do
         }
 
   @conditions [:on_job_success, :on_job_failure, :always, :js_expression]
+
+  @derive {Jason.Encoder,
+           only: [
+             :id,
+             :condition_type,
+             :enabled,
+             :source_job_id,
+             :source_trigger_id,
+             :target_job_id
+           ]}
```

Comment:
I think we also need `condition_expression` and `condition_label` to allow for JS based conditions no?

Reply:
Done